.github/workflows/markdown.yml

```diff
@@ -8,7 +8,7 @@ on:
 jobs:
   test_markdown:
     name: Test markdown syntax
-    runs-on: ubuntu-16.04
+    runs-on: ubuntu-latest
```

Comment:
good spot! wondered why we had any pinned ubuntu releases in Github actions at all.

Reply:
Copy paste from the GHA docs I guess